WcaOnRails/lib/custom_i18n_scanner.rb

```diff
@@ -0,0 +1,76 @@
+require 'i18n/tasks/scanners/file_scanner'
+class WCAFileScanner < I18n::Tasks::Scanners::FileScanner
+  include I18n::Tasks::Scanners::RelativeKeys
+  include I18n::Tasks::Scanners::OccurrenceFromPosition
+
+  def active_record?(model)
+    # FIXME: Find a way to determine this dynamically ?
+    model != "contact"
+  end
+
+  def extract_model_attr(key)
+    model = key.gsub(/^(\w+)\..*$/, '\1').singularize
+    # Some dirty tricks to assign the detected field to the "correct" model
+    if model == "devise"
+      model = "user"
+    elsif model == "admin"
+      model = "person"
+    elsif model == "oauth"
+      model = "doorkeeper/application"
+    end
```

Comment:
Ah, and could you explain what this admin -> person is about. Maybe it's something obvious but I don't know =)